<launch>
  <node pkg="door_tracker" type="door_tracker_node" output="screen" name="door_tracker_node">
    <rosparam command="load" ns="filters" file="$(find laser_scan)/default_scan_shadows.yaml" />  
    <param name="p_fixed_frame" value="odom"/>
    <param name="publish_all_candidates" value="true"/>
    <param name="p_door_min_width" value="0.8"/>
    <param name="p_door_max_width" value="1.0"/>
  </node>
  <node pkg="door_tracker" type="door_database_node" output="screen" name="door_database_node"/>

</launch>